src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1029,8 +1030,8 @@ def _validate_and_get_connection_string(resource_group_name, storage_account):
         raise CLIError(error_message)
 
     keys = storage_client.storage_accounts.list_keys(resource_group_name, storage_account).keys
```

Comment:
@aspaan, in the next PR, please update the message like Donna suggested, or you can choose to do it when she goes through all of the text